### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by jburel**
Update for figure.o.org
cc @will-moore

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -192,14 +197,14 @@ <h3>Upgrade the app</h3>
             </li>
             <li>Install the app using <a href="https://pip.pypa.io/en/stable/">pip</a>:<br>
               <code style="width:650px">
-                $ pip install omero-figure
+                $ pip install -U omero-figure
```
Comment (will-moore) [inline index.html:200]: I think we can remove this whole section of "upgrade from a version of OMERO.figure prior to 2.0.1" since that is quite old now.

**Comment by will-moore** [comment]
Thanks for the update.
Can probably remove the sentence "OMERO.figure version 2.x requires OMERO.web 5.2.6 or newer." It's still true but seems kinda dated.

**Comment by jburel** [comment]
@will-moore I have just also noticed we still point to jcb viewer. We will need to sort that out
Maybe time to redirect figure.o.org to another page e.g. www.o.org

**Comment by will-moore** [comment]
Yes, I think we can just redirect to https://www.openmicroscopy.org/omero/figure/ now and save ourselves the maintenance of the old site.

**Pull request closed without merging**